src/users.cpp

```diff
@@ -1227,11 +1227,11 @@ ConnectClass::ConnectClass(std::shared_ptr<ConfigTag> tag, char t, const std::ve
 }
 
 ConnectClass::ConnectClass(std::shared_ptr<ConfigTag> tag, char t, const std::vector<std::string>& masks, std::shared_ptr<ConnectClass> parent)
+	: hosts(masks)
```

Comment:
This is wrong because the call to Update() might change the hosts field even though we want the value passed in the constructor to be used.